packages/analytics-js-integrations/src/integrations/Webengage/nativeSdkLoader.js

```diff
@@ -0,0 +1,46 @@
+import { LOAD_ORIGIN } from '@rudderstack/analytics-js-common/v1.1/utils/constants';
+
+function loadNativeSdk(licenseCode, dataCentre, window) {
+  var webengage;
+  !(function (w, e, b, n, g) {
```

Comment:
add eslint and sonar start/stop magic comments to exclude the loading snippet of the third party from checks

Reply:
@bardisg These statements are not needed here as all `nativeSdkLoader.js` files are excluded in the config.